content/blog/2025-10-20-announcing-btrfs-support-in-almalinux-10-1.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Announcing Btrfs support in AlmaLinux OS 10.1"
+type: blog
+author:
+  name: "Davide Cavalca"
+  bio: "Production Engineer, Linux Userspace @ Meta"
+  image: /users/davidecavalca.png
+date: "2025-10-20"
+images:
+  - /blog-images/2025/btrfs_almalinux101.png
+post:
+  title: "Announcing Btrfs support in AlmaLinux OS 10.1"
+  image: /blog-images/2025/btrfs_almalinux101.png
+---
+
+# Announcing Btrfs support in AlmaLinux OS 10.1
+
+AlmaLinux OS 10.1 will support the Btrfs filesystem, which has already been available in AlmaLinux OS Kitten since 2025-09-10.
+
+## What is Btrfs?
+
+[Btrfs](https://btrfs.readthedocs.io/) is a modern Copy-on-Write (CoW) filesystem implementing advanced features while also focusing on fault tolerance, repair and easy administration. Among other things, it provides snapshotting, built-in volume management, checksumming for data and metadata, transparent compression and efficient copying via reflinks. These features lead to better performance for most common workloads and stronger resistance against bitrot and other potential issues with the underlying storage medium. Btrfs is already supported by a variety of Linux distributions, and is the default filesystem for several of them, notably Fedora Linux and openSUSE.
```

Comment:
not sure how much details we need here, but not all Fedora editions default to Btrfs and given the audience IDK if Server people will notice this